no .git directory in tarball, disable failure on gradle-git-properties plugin

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -639,6 +639,7 @@ val gitPropertiesFile = tasks.named<com.gorylenko.GenerateGitPropertiesTask>("

 gitProperties {
 	dateFormat = "yyyy-MM-dd'T'HH:mm:ssX"
+	failOnNoGitDirectory = false
 }

 // gradle-git-properties adds its output dir to the main resources, so every Jar
